addons/website_sale/views/templates.xml

```diff
@@ -2279,6 +2279,18 @@
                     </div>
                     <t t-elif="not hide_payment_button" t-call="payment.submit_button"/>
                 </t>
+                <t t-elif="xmlid == 'website_sale.address'">
+                    <a
+                        role="button"
+                        name="website_sale_main_button"
+                        id="save_address"
+                        t-attf-class="#{_cta_classes} btn btn-primary #{not website_sale_order._is_cart_ready() and 'disabled'} #{_form_send_navigation and 'order-lg-3 w-100 w-lg-auto ms-lg-auto' or 'w-100'}"
+                        t-att-href="step_specific_values['main_button_href']"
+                    >
+                        <t t-out="step_specific_values['main_button']"/>
+                        <i class="fa fa-angle-right ms-2 fw-light"/>
+                    </a>
+                </t>
```

Comment:
Let's avoid duplicating code :)

Worst case scenario you can add something like this to add an id only if `xmlid == 'website_sale.address'`
```xml
t-att="['id', 'save_address'] if xmlid == 'website_sale.address' else []"
``` 
```suggestion
```